packages/rum-core/src/boot/startRum.ts

```diff
@@ -61,7 +62,11 @@ export function startRum(
     lifeCycle.notify(LifeCycleEventType.RAW_ERROR_COLLECTED, { error })
   }
   if (!canUseEventBridge()) {
-    startRumBatch(configuration, lifeCycle, telemetry.observable, reportError)
+    const pageExitObservable = createPageExitObservable()
+    pageExitObservable.subscribe((event) => {
+      lifeCycle.notify(LifeCycleEventType.PAGE_EXITED, event)
+    })
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏Do we have to create a lifeCycle event if we have the observable?

Reply:
I'm not quite against it, but we introduced the `LifeCycle` specifically to avoid passing a bunch of individual `Observable` all around. Using the `LifeCycle` allows to have a central event pub/sub in RUM and simplifies writing tests. Using the `Observable` would be a step backward from this design. I'd prefer to keep using the `LifeCycle` for now. @bcaudan WDYT?

Reply:
I do get the convenience of the LifeCycle but then could we also use it in `startRumBatch`. It's not pretty straightforward to understand why we sometimes pass down the observable and not other times. Now that both RUM and Logs have a lifecycle we could add the PAGE_EXITED event to Logs and use it in the batch. Wdyt?

Reply:
FTR: we agreed that this topic requires further discussion. I'll merge this as-is and we'll continue the discussion around an RFC